tools/download_bootswatch_fonts.R

```diff
@@ -31,7 +31,18 @@ download_and_copy_fonts <-  function(theme) {
   urls <- sass:::extract_group(css, "url\\(([^)]+)")
   basenames <- basename(urls)
   Map(function(url, nm) {
-    sass:::download_file(url, file.path(fonts_home, nm))
+    target <- file.path(fonts_home, nm)
+    # The basename can sometimes be very long, and R CMD check
+    # will complain if the target file is over 100 characters long,
+    # so shorten it if necessary
+    if (nchar(target) > 100) {
+      nm <- paste0(
+        rlang::hash(tools::file_path_sans_ext(nm)),
```

Comment:
@cpsievert re-hashing the file name is a neat trick! Do the long hashed file names come from Google Fonts?